### PR TITLE
Bumping version to 0.6.4

### PR DESCRIPTION
**Pull request by izgeri**
#### What is the status of the manual tests?
Have you run the following manual tests to verify existing functionality continues to function as expected?
- [x] Manually tested [K8s CRDs](https://github.com/cyberark/secretless-broker/tree/master/test/manual/k8s_crds)
- [x] Manually tested [Keychain provider](https://github.com/cyberark/secretless-broker/tree/master/test/manual/keychain_provider)
- [x] Manually run the [K8s demo](https://github.com/cyberark/secretless-broker/tree/master/demos/k8s-demo)
- [ ] Manually run [Kubernetes-Conjur demo](https://github.com/conjurdemos/kubernetes-conjur-demo) with a local Secretless Broker image build of your branch
- [ ] Manually run the [full demo](https://github.com/cyberark/secretless-broker/tree/master/demos/full-demo) (optional)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -234,7 +234,7 @@ _Please note: Plugin API interface signatures and supported plugin API version(s
 
 1. Based on the unreleased content, determine the new version number and update
    the [VERSION](VERSION) file.
-1. Run `./bin/prefill_changelog` to populate the [changelog](CHANGELOG.md) with
+1. Run `./bin/prefill_changelog $(cat VERSION)` to populate the [changelog](CHANGELOG.md) with
```
Comment (doodlesbykumbi) [inline CONTRIBUTING.md]: @izgeri there's a danglingly cat

Comment (izgeri) [reply CONTRIBUTING.md]: I'm not sure I follow - the command you run is `./bin/prefill_changelog [version]`, and `$(cat VERSION)` echos the `[version]` to that command

Comment (izgeri) [reply CONTRIBUTING.md]: so this change updates the contributing file to make it easier to run the changelog filler

Comment (doodlesbykumbi) [reply CONTRIBUTING.md]: 🤦‍♂️ I parsed this completely wrong

**Pull request merged**